superset-frontend/src/views/CRUD/data/database/DatabaseModal/ModalHeader.tsx

```diff
@@ -142,19 +154,23 @@ const ModalHeader = ({
     </StyledFormHeader>
   );
 
+  const importDbHeader = (
+    <StyledStickyHeader>
+      <StyledFormHeader>
+        <p className="helper-top"> STEP 2 OF 2 </p>
+        <h4>Enter the required {dbModel.name} credentials</h4>
+        <p className="helper-bottom">{fileCheck ? file[0].name : ''}</p>
+      </StyledFormHeader>
+    </StyledStickyHeader>
+  );
+
+  if (fileCheck) return importDbHeader;
   if (isLoading) return <></>;
```

Comment:
`<></>;` looks like a fish

Reply:
:) aww.. with a hook in its mouth.

Reply:
Hahaha we can call it the loading fish! 🎣